cypress/integration/identity_creation.spec.js

```diff
@@ -71,6 +71,12 @@ context("identity creation", () => {
 
           // We should land back on the intro screen
           cy.get('[data-cy="get-started-button"]').should("exist");
```

Comment:
You wanna be on the second screen when testing wether the `esc` key works.

```suggestion
          cy.get('[data-cy="get-started-button"]').click();
```